src/main/java/com/amazon/sqs/javamessaging/message/SQSMessage.java

```diff
@@ -111,13 +113,13 @@ public class SQSMessage implements Message {
      * This is called at the receiver side to create a
      * JMS message from the SQS message received.
      */
-    SQSMessage(Acknowledger acknowledger, String queueUrl, com.amazonaws.services.sqs.model.Message sqsMessage) throws JMSException{
+    SQSMessage(Acknowledger acknowledger, String queueUrl, software.amazon.awssdk.services.sqs.model.Message sqsMessage) throws JMSException{
```

Comment:
Why not import message?

Reply:
It is because we also have `javax.jms.Message` in this context. I can use `as` if that's preferred.

Reply:
I was wrong about import `as` comment^. There's no import aliasing in Java. Will keep as it is.